Airports/EGKK/Positions.txt

```diff
@@ -3,6 +3,11 @@ EGKK-S_APP:Gatwick Director:126.820:KKS:S:EGKKS:APP:-:-:3750:3761:N051.08.53.000
 EGKK-S_APP:Gatwick Director:126.820:KKS:S:EGKK-S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK-S_APP:Gatwick Director:126.820:KKS:S:EGKK+S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK-S_APP:Gatwick Director:126.820:KKS:S:EGKK~S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKKS:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK-S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK+S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK~S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
```

Comment:
```suggestion
EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
EGKK-S_APP:Gatwick Director:129.020:KKS:S:EGKK-S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
```

These two options will be enough, since the only reason for all the whacky logon callsigns is because INT & SPT share the same frequency.